common/beerocks/bwl/dummy/ap_wlan_hal_dummy.cpp

```diff
@@ -379,9 +379,11 @@ bool ap_wlan_hal_dummy::process_dummy_event(parsed_obj_map_t &parsed_obj)
             LOG(ERROR) << "Failed reading mac parameter!";
             return false;
         }
+        msg->params.mac  = beerocks::net::network_utils::mac_from_string(tmp_str);
+        char assoc_req[] = "dummy association frame";
```

Comment:
I think it would be better to include a real association frame here, e.g. something that you got from sniffing actual easymesh TLVs from a certified device.

Reply:
done.